### DISCUSSION TITLE
Coarse font-weight buckets?

### DISCUSSION THREAD

**Discussion by sccmcca** [squidfunk/mkdocs-material#3593] [Q&A] [answered]
Hi all,

I'm trying to adjust the font-weight for some of my text at a precise value of 450. However, it looks like only buckets of 100 get rendered. For example, a value of 449 seems to round down and render as 400, and a value of 450 seems to round up and render as 500. 

Any ideas how I can override this to use my exact font-weight values? Thanks!

**Comment by squidfunk** [comment, marked as answer]
Not supported for non-variable fonts, please see https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight. There are [variable fonts](https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight#variable_fonts) out there that might support this though.
